src/studio/src/designer/DataModeling.Tests/_TestData/Model/JsonSchema/SeresSimpleTypeRestrictions.json

```diff
@@ -94,15 +98,40 @@
     "f6"
   ],
   "$defs": {
+    "SeresType": {
+      "allOf": [
+        {
+          "type": "string",
+          "@xsdType": "string"
+        },
+        {
+          "enum": [
+            "frontend",
+            "backend",
+            "other"
+          ]
+        }
```

Comment:
Should there be an @xsdUnhandledEnumAttributes here to match the xsd?

Reply:
Yep, good point. Done.